packages/ui/webpack.config.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
This looks strange here. Typescript rule for js file?

Reply:
It looks that just my IDE complains about it.